components/advanced-search/search-filters.tsx

```diff
@@ -181,6 +185,29 @@ const SearchFilters: React.FC<{
             options={getNaturesJuridiques()}
           />
         </div>
+        <SimpleSeparator />
+        <div>
+          <label>Effectif global</label>
+          <MultiSelect
+            name="tranche_effectif_salarie"
+            defaultValue={tranche_effectif_salarie}
+            placeholder="Choisir une tranche d'effectif"
+            id="effectif-salarie-multi-select"
+            instanceId="effectif-salarie-multi-select"
+            options={tranchesEffectifs}
+          />
+        </div>
+        <div>
+          <label>Catégorie d&apos;entreprise</label>
+          <MultiSelect
+            name="categorie_entreprise"
+            defaultValue={categorie_entreprise}
+            placeholder="Choisir une catégorie d'entreprise"
+            id="categorie-entreprise-multi-select"
+            instanceId="categorie-entreprise-multi-select"
+            options={categoriesEntrepriseFilter}
```

Comment:
Maybe prefer `categoriesEntrepriseFilterOptions` and `tranchesEffectifsOptions` for consistency ?